src/js/types/search.ts

```diff
@@ -0,0 +1,52 @@
+export interface SearchFieldResponse {
+  sections: Section[];
+}
+
+export interface Section {
+  fields: Field[];
+  section_title: string;
+}
+
+export interface Field {
+  config: Config;
+  datatype: string;
+  description: string;
+  id: string;
+  mapping: string;
+  options: string[];
+  title: string;
+  mapping_for_search_filter?: string;
+}
+
+export interface Config {
+  bin_size?: number;
+  maximum?: number;
+  minimum?: number;
+  taper_left?: number;
+  taper_right?: number;
+  units?: string;
+  enum?: null;
```

Comment:
I think there is a type mistake here, enum will only be undefined or null this way

Reply:
done!